frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/HIVAoEForm.tsx

```diff
@@ -58,17 +59,28 @@ export const HIVAoEForm = ({
         </div>
       </div>
       <div className="grid-row">
-        <div className="tablet:grid-col-6">
+        <div className="tablet:grid-col-12">
           <MultiSelect
             name={`sexual-partner-gender-${testOrder.internalId}`}
             options={GENDER_IDENTITY_VALUES}
             onChange={onSexualPartnerGenderChange}
             initialSelectedValues={selectedGenders}
-            label={"What is the gender of their sexual partners?"}
+            label={
+              <>
+                What is the gender of their sexual partners?{" "}
+                <span className={"text-base-dark"}>
+                  (Select all that apply.)
+                </span>
+              </>
+            }
+            dropdownClassName={""}
```

Comment:
Could you please explain to me what the `dropdownClassName ` is doing/why we need it? In the props it's defined as optional, and then the only place I ever see it being set is here as an empty string. Then when it's value is passed in the common component file you pass it with a nullish coalescing operator. Is there a case where it ever won't  be null or an empty string? If so, why not just get rid of it?

Reply:
This is primarily being used to avoid having the `multi-select-dropdown` class be applied to the combo box since that removes the max-width style and causes it to stretch to all the available width. In the form here we allow the full 12 column slots in `grid-col-12` so that the label and tooltip/hint text don't wrap onto a new line, but we don't want to allow the combo box to use that full width. I put it as a prop like this so that you can choose to override the dropdown className entirely as opposed to just adding on an additional class, but now that I think about it a bit more I do think the latter is more consistent and I can just explicitly pass in an override to the max-width so I'll make an update to it

![image](https://github.com/CDCgov/prime-simplereport/assets/23287037/84769913-9d7f-4333-9028-6536a0cca1e6)